compiler-rt/lib/sanitizer_common/symbolizer/scripts/build_symbolizer.sh

```diff
@@ -88,10 +89,11 @@ make -j libz.a
 # Build and install libcxxabi and libcxx.
 if [[ ! -f ${LLVM_BUILD}/build.ninja ]]; then
   rm -rf ${LIBCXX_BUILD}
```

Comment:
Is this intentional, not rm LIBCXX_INSTALL ?

Reply:
Oh yes, missed that one.